pkg/operator/controllers/muo/muo_controller.go

```diff
@@ -34,7 +34,7 @@ const (
 	controllerEnabled                = "rh.srep.muo.enabled"
 	controllerManaged                = "rh.srep.muo.managed"
 	controllerPullSpec               = "rh.srep.muo.deploy.pullspec"
-	controllerAllowOCM               = "rh.srep.muo.deploy.allowOCM"
+	controllerDisableOCM             = "rh.srep.muo.deploy.disableOCM"
```

Comment:
Q: for clarity, this flag should not be on clusters yet so changing this won't leave an unused value laying around, correct?

FWIW I liked how it read before, especially once we use it on line 100 where we were avoiding a double negative and now have to read "not disabled" vs "allowed"

Reply:
This flag is for if we need to turn it off, since we still need some control (for local updates), but it is a snowflake case and not the default.

Reply:
All of our feature flags use lowercase names and `.enabled = true` and we should definitely be consistent for all of them. 

I would suggest using  `rh.srep.muo.deploy.ocm.enabled` or more simply `rh.srep.muo.ocm.enabled`, which could also be extended to the OCM BaseUrl naming; `rh.srep.muo.ocm.baseurl`.

Reply:
This runs into the issue where a missing flag means false, but you want the feature to be on by default. I'll tweak the naming a bit.